package.json

```
@@ -9,7 +9,7 @@
     "lint": "eslint",
     "lint:fix": "eslint --fix",
     "test": "npm run test:unit && npm run test:typescript",
-    "test:unit": "tap",
+    "test:unit": "borp -C --check-coverage --lines 95 --statements 95 --branches 95 --reporter=@jsumners/line-reporter",
```

```suggestion
    "test:unit": "borp -C --check-coverage --lines 100 --reporter=@jsumners/line-reporter",
```

If you can't reach 100% test coverage, write a ignore comment with clear explanations ` /* c8 ignore start `, we should have very good reasons to ignore coverage.

---

I 've discovered what moved the coverage from 100 to 99

[This test case](https://github.com/SamSalvatico/under-pressure/blob/736d71c9af0c6b26644bf0337f853d9464b5a667/test/pressurehandler.test.js#L255)

[Tap Version](https://github.com/fastify/under-pressure/blob/master/test/pressurehandler.test.js#L271)

In the tap one the code [falls as elu is undefined](https://github.com/SamSalvatico/under-pressure/blob/736d71c9af0c6b26644bf0337f853d9464b5a667/index.js#L165)
but in the sinon one not

I spent a few hours trying to understand how to mock it correctly
Does anybody have some idea about that?

---

In my opinion: skip that sinon nonsense.

```js
const origMethod = perfHooks.monitorEventLoopDelay
perfHooks.monitorEventLoopDelay = (...args) => {
	console.log('delay:', args)
	return origMethod.apply(perfHooks, args)
}

t.after(() => {
	perfHooks.monitorEventLoopDelay = origMethod
})
```

---

removed sinon, but I has not been able to make it work without proxyquire
https://github.com/fastify/under-pressure/pull/253/commits/fdf0e5b5370e99b079025b373c95a0fcdfdb14cb#diff-8be66950450e9b9cb0762c3a5211b0e2ddfcc8e5d0c2693d054414cac3ab066aR258

---

That makes since because `node:` prevents utilizing the require cache. But I have not had this problem in other tests 🤔